templates.yaml

```diff
@@ -45,3 +45,44 @@ ag_news:
       {{["Not-Related", "Related"][label]}}'
     name: Test1
     reference: ''
+imdb:
+  imdb_1: !Template
+    jinja: '{{text}}
+
+      What is the sentiment expressed in this text?
+
+      {{ ["neg", "pos"] [label] }}'
```

Comment:
Use words or english phrases as opposed to abbreviations "negative" "positive" etc.

Reply:
Done